src/overlays/actors/ovl_En_Test7/z_en_test7.c

```diff
@@ -432,184 +419,188 @@ void EnTest7_Destroy(Actor* thisx, PlayState* play) {
 }
 
-void func_80AF19A8(EnTest7* this, PlayState* play) {
+void EnTest7_StartWarpCs(EnTest7* this, PlayState* play) {
     if (!CutsceneManager_IsNext(play->playerCsIds[PLAYER_CS_ID_SONG_WARP])) {
         CutsceneManager_Queue(play->playerCsIds[PLAYER_CS_ID_SONG_WARP]);
     } else {
         CutsceneManager_Start(play->playerCsIds[PLAYER_CS_ID_SONG_WARP], NULL);
-        func_80AF082C(this, func_80AF1A2C);
+        EnTest7_SetupAction(this, EnTest7_WarpCsPart1);
         play->unk_18844 = true;
     }
 }
 
-void func_80AF1A2C(EnTest7* this, PlayState* play) {
+void EnTest7_WarpCsPart1(EnTest7* this, PlayState* play) {
     Color_RGB8 fogColor = { 64, 0, 0 };
     Color_RGB8 ambientColor = { 220, 220, 255 };
-    f32 envLerp = this->unk_1E54 / 10.0f;
+    f32 envLerp = this->timer / (f32)(10 - 0);
 
     Environment_LerpAmbientColor(play, &ambientColor, envLerp);
     Environment_LerpFogColor(play, &fogColor, envLerp);
     Environment_LerpFog(play, 2000, 4000, envLerp);
 
-    if (this->unk_1E54 >= 10) {
+    if (this->timer >= 10) {
         Camera* subCam =
             Play_GetCamera(play, CutsceneManager_GetCurrentSubCamId(play->playerCsIds[PLAYER_CS_ID_SONG_WARP]));
 
         this->subCamEye = subCam->eye;
         this->subCamAt = subCam->at;
         this->subCamFov = subCam->fov;
 
-        func_80AF082C(this, func_80AF1CA0);
-        this->unk_144 |= 0x20;
+        EnTest7_SetupAction(this, EnTest7_WarpCsPart2);
+        this->flags |= OWL_WARP_FLAGS_20;
         Audio_PlaySfx_AtPos(&this->actor.projectedPos, NA_SE_PL_WARP_WING_OPEN);
         Play_EnableMotionBlur(120);
     }
 }
 
 void func_80AF1B68(EnTest7* this, PlayState* play) {
-    this->unk_144 |= 2;
+    this->flags |= OWL_WARP_FLAGS_DRAW_WIND_CAPSULE;
 
-    if (this->unk_148.unk_04 < 11.0f) {
-        this->unk_148.unk_04 = this->unk_148.unk_04 + 0.3f;
-        this->unk_148.unk_08 = ((this->unk_148.unk_00 * -0.45f) / 11.0f) + 0.7f;
-        this->unk_148.unk_0C = ((this->unk_148.unk_00 * -0.29999998f) / 11.0f) + 0.7f;
+    if (this->windCapsule.unk_04 < 11.0f) {
+        this->windCapsule.unk_04 += 0.3f;
+        this->windCapsule.xzScale = ((this->windCapsule.unk_00 * -0.45f) / 11.0f) + 0.7f;
+        this->windCapsule.yScale = ((this->windCapsule.unk_00 * -0.29999998f) / 11.0f) + 0.7f;
     }
 
-    if (this->unk_148.unk_00 < 11.0f) {
-        this->unk_148.unk_00 += 1.0f;
-        if (this->unk_148.unk_00 > 6.0f) {
+    if (this->windCapsule.unk_00 < 11.0f) {
+        this->windCapsule.unk_00 += 1.0f;
+        if (this->windCapsule.unk_00 > 6.0f) {
             Player* player = GET_PLAYER(play);
 
-            this->unk_144 &= ~1;
+            this->flags &= ~OWL_WARP_FLAGS_DRAW_WINGS;
             player->actor.draw = NULL;
         }
-    } else if (this->unk_1E54 >= 87) {
-        func_80AF082C(this, func_80AF1F48);
-        this->unk_144 &= -9;
-        this->unk_148.unk_10 -= 0x2EE0;
+    } else if (this->timer >= 87) {
```

Comment:
```suggestion
    } else if (this->timer > 86) {
```
There is a function below that actually does `this->timer - 86` so seems nice to reference 86 here as well.